### ISSUE TITLE
Any specific reason why test is done one image at a time instead of batch?

### ISSUE THREAD

**Issue by anjandeepsahni** [ShuangXieIrene/ssds.pytorch#34]
Hi,

I was reviewing the test code here, and found out that test_epoch defined in ssds_train.py tests one image at a time. Why can't we do this in batch mode (for example as in eval_epoch)?

**Comment by blueardour** [comment]
Hi, I'm new in the training and not able to give authority answers. However, in my opinion, it doesn't matter whether the batch size equals 1 or some one bigger.  It does affect the runtime speed, as a bulk of data may result better GPU utilization. The final accuracy would be all the same.
